c-dependencies/js-compute-runtime/xqd.h

```diff
@@ -54,6 +54,49 @@ typedef struct {
 
 #define INVALID_HANDLE (UINT32_MAX - 1)
 
+typedef enum FastlyStatus {
+  // Success value.
+  // This indicates that a hostcall finished successfully.
+  ok = 0,
```

Comment:
I'd be inclined to capitalize the enum values.
```suggestion
  Ok = 0,
```